node/hyperbahn/service_proxy.js

```diff
@@ -424,6 +433,63 @@ function isExitFor(serviceName) {
     return chan.serviceProxyMode === 'exit';
 };
 
+ServiceDispatchHandler.prototype.handleCircuitStateChange =
+function handleCircuitStateChange(change) {
+    var self = this;
+
+    var circuit = change.circuit;
+    var oldState = change.oldState;
+    var state = change.state;
+
+    if (oldState && oldState.healthy !== state.healthy) {
+        // unhealthy -> healthy
+        if (state.healthy) {
+            self.statsd.increment('circuits.healthy.total', 1);
```

Comment:
This code is in hyperbahn and doesn't have access to any of the fast stat code in tchannel. For loose coupling we are just going to hard code statsd. 

We could pull fast stats into a seperate library but that will take more work thats out of scope for this PR.

:+1: for metrics doc though.